packages/server/lib/browsers/cdp_automation.ts

```diff
@@ -191,8 +191,10 @@ export class CdpAutomation implements CDPClient {
   static async create (sendDebuggerCommandFn: SendDebuggerCommand, onFn: OnFn, sendCloseCommandFn: SendCloseCommand, automation: Automation, protocolManager?: ProtocolManagerShape): Promise<CdpAutomation> {
     const cdpAutomation = new CdpAutomation(sendDebuggerCommandFn, onFn, sendCloseCommandFn, automation)
 
+    // TODO: clean this up. post data size should be populated not the other ones
```

Comment:
Does this TODO still need to be resolved?

Reply:
No it doesn't need to be resolved anymore. Good catch: [`580485e` (#27462)](https://github.com/cypress-io/cypress/pull/27462/commits/580485e928fa4e3588e6720359dcb38f7a0aaa92)